src/screens/UserPortal/Events/Events.spec.tsx

```
@@ -1425,28 +1448,27 @@ describe('Testing Events Screen [User Portal]', () => {
     // Should handle null values without crashing
-    await waitFor(() => {
-      expect(screen.getByTestId('eventStartAt')).toBeInTheDocument();
-    });
+    expect(screen.getByTestId('eventStartAt')).toBeInTheDocument();
   });
 
   it('Should handle network error gracefully', async () => {
     const consoleWarnSpy = vi
       .spyOn(console, 'warn')
-      .mockImplementation(() => {});
+      .mockImplementation(() => { });
```

ESLint

[error] Delete `·`

(prettier/prettier)